modules/hitlnext/src/backend/repository.ts

```diff
@@ -428,13 +441,14 @@ export default class Repository {
 
   updateHandoff = async (botId: string, id: string, attributes: Partial<IHandoff>) => {
     const now = new Date()
-    const payload = this.castDate(
+    let payload = this.castDate(
```

Comment:
not a fan of `cast` here either as you don't cast, you serialize. I know I probably questioned it already but aren't date serializable by default ?

Reply:
Agreed method name must change. This method uses `this.bp.database.date.format()` to serialize dates.